lib/ansible/modules/windows/win_dns_record.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Hitachi ID Systems, Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# This is a windows documentation stub. The actual code lives in the .ps1
+# file of the same name.
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: win_dns_record
+version_added: "2.8"
+short_description: Manage Windows Server DNS records
+description:
+- Manages DNS records within an existing Windows Server DNS zone
+author: John Nelson (@johnboy2)
+options:
+  name:
+    description:
+    - The name of the record.
+    required: yes
+    type: str
+  state:
+    description:
+    - Whether the record should exist or not.
+    choices: [ absent, present ]
+    default: present
+    type: str
+  ttl:
+    description:
+    - The "time to live" of the record, in seconds. Ignored when
```

Comment:
Use multiple lines. The first line explains the parameter briefly.
Subsequent lines clarify when to use it, how to use it, what values are expected etc.
```suggestion
    - The "time to live" of the record, in seconds.
    - Ignored when
```